ui/src/onboarding/containers/LoginPageContents.tsx

```diff
@@ -61,7 +61,8 @@ class LoginPageContents extends PureComponent<DispatchProps> {
 
   public async componentDidMount() {
     try {
-      const config = await getAuth0Config()
+      const redirectTo = window.localStorage.getItem('redirectTo') || '/'
```

Comment:
the feature flag makes more sense here i think - here it can control what gets sent to `getAuth0Config` so that that layer doesn't even need to be aware of the flag. something like:

```ts
let config
if (isFlagEnabled('redirectto')) {
  const redirectTo = window.localStorage.getItem('redirectTo') || '/'
  config = await getAuth0Config(redirectTo)
} else {
  config = await getAuth0Config()
}
```
doing it this way (not sending an empty string will leave the url a little cleaner - it won't have `?redirectTo=` with no value in it - in other words, only append `?redirectTo=${actualURL}` if there's something to redirect to.

Reply:
That makes sense